.github/workflows/ckan_auto.yml

```diff
@@ -4,10 +4,11 @@ name: 4 - Automated CKAN Jobs
 on:
   schedule:
     - cron: '30 7 * * *'       # Tracking Update -- every day at 2:30am EST
-    - cron: '0 2 * * *'        # S3 Sitemap Update -- every day at 7pm EST
+    - cron: '0 0 * * *'        # S3 Sitemap Update -- every day at 7pm EST
     - cron: '4/15 * * * *'     # Harvester Check -- every 15 mins
     - cron: '0 3 * * *'        # DB-Solr-Sync -- every day at 10pm EST
     - cron: '30 6 * * *'       # Check Stuck Jobs -- every day at 1:30am EST
+    - cron: '0 13 * * *'       # Generate Reports -- every day @9:00am EST
```

Comment:
Refreshing report cache at 9AM EST is debatable. I can think a few pros and cons about 9AM but let us go with it for now.